compiler/rustc_codegen_ssa/messages.ftl

```diff
@@ -22,6 +22,8 @@ codegen_ssa_copy_path_buf = unable to copy {$source_file} to {$output_path}: {$e
 
 codegen_ssa_create_temp_dir = couldn't create a temp dir: {$error}
 
+codegen_ssa_dangerous_stack_allocation = Dangerous stack allocation of size: {$output} exceeds most architecture limits
```

Comment:
I would tweak this message to explain a bit what "dangerous" means or not use that word at all. For instance:

```suggestion
codegen_ssa_dangerous_stack_allocation = Large stack allocation ({$output}) may exceed the stack available and terminate the program
```